addons/dogtags/CfgUIGrids.hpp

```diff
@@ -0,0 +1,20 @@
+class CfgUIGrids {
+    class IGUI {
+        class Presets {
+            class Arma3 {
+                class Variables {
+                    grid_ACE_dogtags[] = {{"(safeZoneX + safeZoneW) - (12.9 * ((safeZoneW / safeZoneH) min 1.2) / 40)", "safeZoneY + 0.175 * safeZoneH", "8 * (((safeZoneW / safeZoneH) min 1.2) / 40)", "8 * ((((safeZoneW / safeZoneH) min 1.2) / 1.2) / 25)"}, "(((safeZoneW / safeZoneH) min 1.2) / 40)", "((((safeZoneW / safeZoneH) min 1.2) / 1.2) / 25)"};
```

Comment:
Suggestion:
`#include "\a3\ui_f\hpp\defineCommonGrids.inc"`
Then this line becomes:
```sqf
GVAR(grid)[] = {
    {
        (safeZoneX + safeZoneW) - 12.9 * GUI_GRID_W,
        safeZoneY + 0.175 * safeZoneH,
        8 * GUI_GRID_W,
        8 * GUI_GRID_H
    },
    GUI_GRID_W,
    GUI_GRID_H
};
```

Reply:
Done